src/pages/sections.conf.js

```diff
@@ -1,69 +1,54 @@
+        key: 'validationRulesAnalysis',
         path: '/validation-rules-analysis',
         component: ValidationRulesAnalysis,
         info: {
-            label: () => i18n.t('Validation Rule Analysis'),
+            label: i18n.t('Validation Rule Analysis'),
             icon: 'done_all',
-            description: () =>
-                i18n.t(
-                    'Run validation rules in order to unveil anomalies and errors in the data in the database.'
-                ),
-            actionText: () => i18n.t('Run Validation'),
+            description: i18n.t(
+                'Run validation rules in order to unveil anomalies and errors in the data in the database.'
+            ),
+            actionText: i18n.t('Run Validation'),
             docs: 'validation_rule_analysis',
         },
     },
     {
-        key: OUTLIER_DETECTION_SECTION_KEY,
+        key: 'outlierDetection',
         path: '/outlier-detection',
         component: OutlierDetection,
         info: {
-            label: () => i18n.t('Outlier detection'),
+            label: i18n.t('Outlier detection'),
             icon: 'show_chart',
-            description: () =>
-                i18n.t(
-                    'Analyze potential outlier values based on standard deviations or min-max values. Outlier values can be examined and marked for follow-up.'
-                ),
-            actionText: () => i18n.t('Analyze'),
+            description: i18n.t(
+                'Analyze potential outlier values based on standard deviations or min-max values. Outlier values can be examined and marked for follow-up.'
+            ),
+            actionText: i18n.t('Analyze'),
             docs: 'outlier_detection',
         },
     },
     {
-        key: FOLLOW_UP_ANALYSIS_SECTION_KEY,
+        key: 'followUpAnalysis',
         path: '/follow-up-analysis',
         component: FollowUpAnalysis,
         info: {
-            label: () => i18n.t('Follow-Up Analysis'),
+            label: i18n.t('Follow-Up Analysis'),
             icon: 'description',
-            description: () =>
-                i18n.t(
-                    'View or edit data values marked for further follow-up during data entry or analysis.'
-                ),
-            actionText: () => i18n.t('View Data Values'),
+            description: i18n.t(
+                'View or edit data values marked for further follow-up during data entry or analysis.'
+            ),
+            actionText: i18n.t('View Data Values'),
             docs: 'follow_up_analysis',
         },
     },
 ]
 
-export const getDocsKeyForSection = sectionKey => {
-    for (let i = 0; i < sections.length; i++) {
-        const section = sections[i]
-        if (section.key === sectionKey) {
-            return section.info.docs
-        }
-    }
-
-    return ''
-}
+export const getDocsKeyForSection = sectionKey =>
+    sections.find(section => section.key === sectionKey)?.info.docs || ''
```

Comment:
Quite terse, but I think for readability here I'd split it up in the call to find, and then have a separate return `... || ...`